### DISCUSSION TITLE
How to reopen the MINGW64 window

### DISCUSSION THREAD

**Discussion by benyltam** [community/community#30486] [Actions] [answered]
Dear All,

I am a GitHub beginner and using Visual Studio on Windows 10.  After I install GIT on my Windows, there is an option "Run Git". I check this option and a widow "MINGW64:/Users/Ben Tam" is open.  After I close this window, I don't know how to reopen it.  Could anybody tell me how to reopen the MINGW64 window?

**Comment by naikrovek** [comment, marked as answer]
it should be somewhere in the start menu.  

“start > all apps > git > git bash” perhaps

**Reply by benyltam** [reply]
Thanks for your reply.
